packages/app-desktop/commands/exportDeletionLog.ts

```diff
@@ -0,0 +1,75 @@
+
+import { CommandRuntime, CommandDeclaration } from '@joplin/lib/services/CommandService';
+import { _ } from '@joplin/lib/locale';
+import Logger from '@joplin/utils/Logger';
+import { TargetType } from '@joplin/utils/Logger';
+import shim from '@joplin/lib/shim';
+import Setting from '@joplin/lib/models/Setting';
+import bridge from '../services/bridge';
+import { Stat } from '@joplin/lib/fs-driver-base';
+
+export const declaration: CommandDeclaration = {
+	name: 'exportDeletionLog',
+	label: () => _('Export deletion log'),
+};
+
+// Return log files orderer from oldest to newest, with log.txt being the last one
+const logsOrderedByTimestamp = (logFiles: Stat[]) => {
+	return logFiles.sort((a, b) => {
+		const aTimestamp = a.path.match(/\d+/);
+		const bTimestamp = b.path.match(/\d+/);
+
+		if (!aTimestamp) return 1;
+		if (!bTimestamp) return -1;
+
+		return parseInt(aTimestamp[0], 10) - parseInt(bTimestamp[0], 10);
+	});
+};
```

Comment:
Wouldn't it be the same to sort alphabetically?

Reply:
You are right, thanks for pointing it out, I'm removing this in favour of `sort()`